build/build.cake

```diff
@@ -1,10 +1,10 @@
-#module nuget:?package=Cake.LongPath.Module&version=0.7.0
+#module nuget:?package=Cake.LongPath.Module&version=1.0.1
 
-#addin nuget:?package=Cake.FileHelpers&version=3.2.1
-#addin nuget:?package=Cake.Powershell&version=0.4.8
-#addin nuget:?package=Cake.GitVersioning&version=3.3.37
+#addin nuget:?package=Cake.FileHelpers&version=4.0.1
+#addin nuget:?package=Cake.Powershell&version=1.0.1
+#addin nuget:?package=Cake.GitVersioning&version=3.4.216
 
-#tool nuget:?package=MSTest.TestAdapter&version=2.1.0
+#tool nuget:?package=MSTest.TestAdapter&version=2.2.4
```

Comment:
This specific version is not going to work with our tests. There is a regression on metadata serialization that will be keep our tests from running. There is no **stable** version that fixed this just yet (I'm validating if 2.2.5-preview-20210605-01 fixes it, and it should), so I would downgrade to 2.2.3 for now.

Reply:
Tests didn't fail. Seeing the logs, there's no issues too! So, the downgrade is not needed!

Reply:
This version in the cake build file should match what is used in the unit test projects.
For example: https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/main/UnitTests/UnitTests.UWP/UnitTests.UWP.csproj#L113-L118
I believe it will fail once you update it everywhere. I don't remember if the regression is in the adapter or in the framework, but anyway they should match.

Reply:
Tracking these versions is hard. How about using NuGet's `CVPM` feature to manage package versions centrally!?

Reply:
@Nirmal4G what's the CVPM feature? Do you have a link to info about it?

Reply:
[**C**entrally **V**ersioned **P**ackage **M**anagement (CVPM)](https://github.com/NuGet/Home/wiki/Centrally-managing-NuGet-package-versions)